.travis.yml

```diff
@@ -6,11 +6,38 @@ branches:
     # 'v' followed by a digit, such as for version tags.
     - master
     - /^v\d.*$/
-cache: pip
+cache:
```

Comment:
What is all this stuff?

Reply:
Oops I meant to link this https://github.com/python-desert/desert/pull/59/commits/6e49644df70dce5366f81c663bda710138b74f71#diff-354f30a63fb0907d4ad57269548329e3R18-R35

Where does this list come from?